texto.html

```diff
@@ -4,9 +4,12 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Hello Word</title>
+    <link rel="stylesheet" href="estilo.css">
```

Comment:
Try to not use spanish words when naming files, like 'estilo.css', instead use 'styles.css'